kernel/nativelambda.ml

```diff
@@ -40,7 +40,7 @@ type lambda =
   | Lfix          of (int array * (string * inductive) array * int) * fix_decl
   | Lcofix        of int * fix_decl
   | Lint          of int (* a constant constructor *)
-  | Lmakeblock    of prefix * pconstructor * int * lambda array
+  | Lmakeblock    of prefix * constructor * int * lambda array
```

Comment:
I am surprised that this data is not required to check constructor cumulativity. I am not an expert on the topic, so let's summon @SkySkimmer (who, hopefully, will dig out an unsoundness proof?).

Reply:
Note that this is not changed by this PR, as this data was already ignored.

Last time I worked on univ poly for the VM and `native_compute`, I was told by @mattam82 that constructors were always carrying the same instance as their inductive type. If that is correct, this information is not needed.